tests/framework/installer/ceph_installer.go

```diff
@@ -56,15 +58,42 @@ type CephInstaller struct {
 	T                func() *testing.T
 }
 
-func (h *CephInstaller) CreateCephCRDs() (err error) {
+func (h *CephInstaller) CreateCephCRDs() error {
 	var resources string
 	logger.Info("Creating Rook CRDs")
 
 	resources = h.Manifests.GetRookCRDs()
 
-	_, err = h.k8shelper.KubectlWithStdin(resources, createFromStdinArgs...)
+	var err error
+	for i := 0; i < 5; i++ {
+		if i > 0 {
+			logger.Infof("waiting 10s...")
+			time.Sleep(10 * time.Second)
+		}
+
+		_, err = h.k8shelper.KubectlWithStdin(resources, createFromStdinArgs...)
+		if err == nil {
+			return nil
+		}
+
+		// If the CRD already exists, the previous test must not have completed cleanup yet.
+		// Delete the CRDs and attempt to wait for the cleanup.
+		if strings.Index(err.Error(), "AlreadyExists") == -1 {
```

Comment:
worth making this a helper function?

Reply:
I hope we don't compare strings from error messages very often so I'm not sure we want to create helpers around the string comparison. This comparison was needed since there is a composite error object due to multiple CRDs created at the same time. I thought about changing the implementation so we create one object at a time, but that was more than I was ready to bite off.